src/main/java/seedu/address/ui/cards/Card.java

```diff
@@ -129,9 +152,10 @@ public boolean equals(Object other) {
             return false;
         }
 
+        @SuppressWarnings("unchecked")
```

Comment:
Not very sure how to resolve this portion. Maybe I should a raw type parameterized instead?